common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```
@@ -16,6 +16,21 @@ _multi_class_auto_insert:
 #################################################
 #################################################
  
+cACTION_APMANAGER_UP_NOTIFICATION:
+  _type: class
+  iface_name_length:
+    _type: uint8_t
+    _length_var: True
```

I think that `_length_var` is used only for TLV classes, tlvf is using it to know if it needs to update the length var.
Therefore:
```suggestion
    iface_name_length: uint8_t
```

---

It is a class, so it is ok :)